packages/sources/config/rollup.config.js

```diff
@@ -44,15 +44,30 @@ const plugins = [
         keep_fnames: true
     }),
     postcss({
-        extract: true
+        extract: true,
+        minimize: true
     }),
     json()
 ];
 
 export default rollupConfig(
-    external(externalDeps({ ...dependencies, ...peerDependencies })),
+    external([
+        ...externalDeps({ ...dependencies, ...peerDependencies, ...devDependencies }),
```

Comment:
No need to list all external deps with stars and such, you can do that in `externalDeps` with easier config. 
```
external(externalDeps(
  { ...dependencies, ...peerDependencies, ...devDependencies },
  [ '@data-driven-forms', 'lodash' ]
))
```

Reply:
But this will override the default externals, so shouldn't it be
```jsx
external(externalDeps(
  { ...dependencies, ...peerDependencies, ...devDependencies },
  ['@patternfly', '@redhat-cloud-services',  '@data-driven-forms', 'lodash' ]
))
```
?

Reply:
Oh yeah! You are right